tests/test_admin.py

```diff
@@ -139,6 +152,20 @@ def test_preview_link_draft_object(self):
             poll_content.get_preview_url()
         )
 
+    def test_change_icon_tooltip(self):
+        """
+        Ensure the change list presents correct tooltip when hovering over the change expiry  icon
```

Comment:
A bit pedantic, but this could be reworded as 
```suggestion
        The change list presents the correct tooltip when hovering over the change expiry icon
```